_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,238 @@
+---
+layout: default
+title: Analyze Jaeger trace data 
+parent: Trace analytics
+nav_order: 55
+---
+
+# Analyze Jaeger trace data
+
+Introduced 2.5
+{: .label .label-purple }
+
+The Trace analytics functionality in the OpenSearch Observability plugin now supports Jaeger trace data. If you use OpenSearch as the backend for Jaeger trace data, you can use the Trace analytics built-in analysis capabilities. This provides support for OpenTelemetry (OTEL) formatted trace data.
+
+When you perform trace analytics, you can select from two data sources:
+
+- **Data Prepper** – Data ingested into OpenSearch through Data Prepper.
+- **Jaeger** – Trace data stored within OpenSearch as its backend.
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the capabilities built into Trace Analytics to analyze the error rates and latencies You can also filter the traces and look into the span details of a trace to pinpoint any service issues.
+
+When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the Dashboards.
+
+Jaeger trace data that you can analyze includes span data, as well as service and operation endpoint data. Jaeger span data analysis requires some configuration.
+
+Each time you ingest data for Jaeger, it creates a separate index for that day. The Dashboards will show the current index that has a mapping.
+
+To learn more about Jaeger data tracing, see the [Jaeger](https://www.jaegertracing.io/) open source documentation.
+
+## Data Ingestion Requirements
+
+If you want to see errors in your trace data, you need to set the following ElasticSearch flag to true prior to data ingestion: `--es.tags-as-fields.all=true`.
+
+Jaeger data that is ingested for OpenSearch needs to have the  flag set for errors. If data is not ingested in this format it will not work for trace analytics with OpenSearch.
+
+### About Data ingestion with Jaeger indexes
+
+Trace analytics for non-Jaeger data use OTEL indexes with the naming conventions `otel-v1-apm-span-*` or `otel-v1-apm-service-map*`.
+
+Jaeger indexes follow the naming conventions `jaeger-span-*` or `jaeger-service-*`.
+
+## How to set up OpenSearch to use Jaeger data
+
+To use trace analytics with Jaeger data, you need to configure error capability for use with trace analytics. We provide a sample Docker compose file that contains the required configurations.
```

Comment:
Who is "We" referring to? Suggested rewrite: "A sample Docker compose file with the required configurations is provided below."

Reply:
it was in a style guide that "we" refers to OpenSearch project team. 
will change wording as you suggest

Reply:
I believe we only use 'we' when referring to us, as a team, like when we say, "we recommend". I saw a good statement from Nate B. about this somewhere...